packages/web3-core-method/src/methods/transaction/GetPendingTransactionMethod.js

```diff
@@ -0,0 +1,36 @@
+/*
+    This file is part of web3.js.
+
+    web3.js is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    web3.js is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @file GetPendingTransactionMethod.js
+ * @author Prince Sinha <sinhaprince013@gmail.com>
+ * @date 2018
+ */
+
+import AbstractMethod from '../../../lib/methods/AbstractMethod';
+
+export default class GetPendingTransactionMethod extends AbstractMethod {
```

Comment:
I would rename this method to ``PendingTransactionsMethod`` because the RPC method is called ``eth_pendingTransactions``.

Reply:
I will merge it after the renaming. Great work! thanks! :)

Reply:
okay, cool @nivida. I will do that.